.travis.yml

```diff
@@ -26,3 +36,7 @@ script:
     - OMP_NUM_THREADS=1 tests/src/eigenwave3d
     - python tests/eigenwave3d.py default --output
     - python tests/eigenwave3d.py read
+    - python tests/eigenwave3d.py pluto --execute --nthreads=4
+    - OMP_NUM_THREADS=4 tests/src/eigenwave3d_pluto
+    - python tests/eigenwave3d.py pluto --output
+    - python tests/eigenwave3d.py read
```

Comment:
If I add pybench.py in requirements, it travis will try to download it and fail, should I just mention this in README?

Date: Fri, 11 Sep 2015 08:33:42 -0700
From: notifications@github.com
To: opesci-fd@noreply.github.com
CC: zlcghoul@hotmail.com
Subject: Re: [opesci-fd] Pluto (#45)

In .travis.yml:

> @@ -26,3 +36,7 @@ script:
>      - OMP_NUM_THREADS=1 tests/src/eigenwave3d
>      - python tests/eigenwave3d.py default --output
>      - python tests/eigenwave3d.py read
> +    - python tests/eigenwave3d.py pluto --execute --nthreads=4
> +    - OMP_NUM_THREADS=4 tests/src/eigenwave3d_pluto
> +    - python tests/eigenwave3d.py pluto --output
> +    - python tests/eigenwave3d.py read

Yes, just add the script, not the results folder or pybench itself. Although you might want to add pybench to requirements.txt, just in case. You might also want to add a short entry to the README in tests how to do the auto-tuning. 


―
Reply to this email directly or view it on GitHub.